src/content/components/IterationView/IterationView.tsx

```diff
@@ -227,8 +226,46 @@ export const IterationView: React.FunctionComponent<
     },
     [metas]
   );
+  const heading = React.useMemo(() => {
+    const nextIteration = iterations.getAdjacentIteration(1, props.iteration);
```

Comment:
For current iteration view, we can hide next arrow as it's mostly empty and if really needed can be reached from select iteration drop down.
```

 const nextIteration =
      props.iteration !== props.currentIteration.number
        ? iterations.getAdjacentIteration(1, props.iteration)
        : null;
```